core/src/main/scala/kafka/tools/TestRaftServer.scala

```diff
@@ -466,3 +459,10 @@ object TestRaftServer extends Logging {
   }
 
 }
+
+private[tools] sealed trait RaftEvent
+private[tools] final case class HandleClaim(epoch: Int) extends RaftEvent
+private[tools] case object HandleResign extends RaftEvent
+private[tools] final case class HandleCommit(reader: BatchReader[Array[Byte]]) extends RaftEvent
+private[tools] final case class HandleSnapshot(reader: SnapshotReader[Array[Byte]]) extends RaftEvent
+private[tools] case object Shutdown extends RaftEvent
```

Comment:
Any reason to move it down the file?

Reply:
Yes, I had to move the case classes either into an `object` or as a top level class because otherwise the Scala compiler warns about an unambigious reference (which fails to build in Kafka since Kafka doesn't allow warnings to come through).